src/main/java/org/edumips64/core/Converter.java

```diff
@@ -564,6 +564,7 @@ public static long powLong(int base, long exp) {
     return ret;
   }
 
+  //Determines if a string has only 0's and 1's chars
```

Comment:
Thanks for adding comments, really appreciated! Would you mind using the javadoc conventions for them?